src/lambdas/sendQueryResultsNotification/queueSendResultsReadyEmail.ts

```diff
@@ -0,0 +1,21 @@
+import { createSecureDownloadLink } from './createSecureDownloadLink'
+import { sendSqsMessage } from '../../sharedServices/queue/sendSqsMessage'
+import { PersonalisationOptions } from '../../types/notify/personalisationOptions'
+import { getEnv } from '../../utils/helpers'
+export const queueSendResultsReadyEmail = async (parameters: {
+  downloadHash: string
+  zendeskTicketId: string
+  recipientEmail: string
+  recipientName: string
+}) => {
+  const emailOptions: PersonalisationOptions = {
+    email: parameters.recipientEmail,
+    firstName: parameters.recipientName,
+    zendeskId: parameters.zendeskTicketId,
+    signedUrl: createSecureDownloadLink(parameters.downloadHash)
```

Comment:
do you think we should rename this to secure download link (in notify as well)?

Reply:
Yes, it is a bit misleading. Probably the best time to do it, while we still can!

Reply:
This has been dealt with now